packages/core/@generated/graphql.ts

```diff
@@ -629,6 +629,18 @@ export type StoreAggregateRating = {
   reviewCount: Scalars['Int']['output']
 }
 
+/** Attribute of a given product. */
```

Comment:
Those changes were made when I run `yarn dev`, not sure why it wasn't already on main